op-challenger/cmd/main.go

```diff
@@ -35,47 +33,33 @@ var VersionWithMeta = func() string {
 }()
 
 func main() {
-	args := os.Args
-	if err := run(args, Main); err != nil {
-		log.Crit("Application failed", "err", err)
-	}
-}
-
-type ConfigAction func(log log.Logger, version string, config *config.Config) error
-
-// run parses the supplied args to create a config.Config instance, sets up logging
-// then calls the supplied ConfigAction.
-// This allows testing the translation from CLI arguments to Config
-func run(args []string, action ConfigAction) error {
-	// Set up logger with a default INFO level in case we fail to parse flags,
-	// otherwise the final critical log won't show what the parsing error was.
 	oplog.SetupDefaults()
 
 	app := cli.NewApp()
 	app.Version = VersionWithMeta
 	app.Flags = flags.Flags
 	app.Name = "op-challenger"
-	app.Usage = "Challenge Invalid L2OutputOracle Outputs"
-	app.Description = "A modular op-stack challenge agent for dispute games written in golang."
+	app.Usage = "Play the fault game"
```

Comment:
I don't understand tightly coupling the op-challenger with the fault game if we explicitly have a `fault` subdir and know it is one way of deleting outputs.
Think we should keep this more simple to start with.
```suggestion
	app.Usage = "Challenge outputs"
```

Reply:
Sure, we can change this. I mean it doesn't actually challenge outputs & there's only one current way to challenge outputs.